generate-moz-grammars/generate-mozcpp.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+# This script updates the mozcpp grammar automatically.
+#
+# Usage: ./generate-moz-grammars/generate-mozcpp.sh
+
+# FIXME we need to remove this line once we are going to use
+# the tree-sitter-cpp bindings
+# Get the tree-sitter-cpp submodule version
+TS_CPP_VERSION=`git submodule status tree-sitter-cpp | awk '{ print $1 }'`
+
+# Enter the mozcpp directory
+pushd tree-sitter-mozcpp
+
+# Create tree-sitter-cpp directory
+mkdir -p tree-sitter-cpp
+
+# Enter tree-sitter-cpp directory
+pushd tree-sitter-cpp
+
+# Shallow clone tree-sitter-cpp to a specific revision
+git init
+git remote add origin https://github.com/tree-sitter/tree-sitter-cpp.git
+git fetch --depth 1 origin $TS_CPP_VERSION
+git checkout FETCH_HEAD
+
+# Clone tree-sitter-c repository
+git clone --depth 1 --branch 0.16.1 https://github.com/tree-sitter/tree-sitter-c.git
```

Comment:
Why 0.16.1?

Reply:
Because it depends on https://github.com/tree-sitter/tree-sitter-cpp/blob/master/package.json#L16

Reply:
Instead of cloning the repo, why don't we use `npm` to install the tree-sitter-cpp dependencies? Then the path to tree-sitter-c will just be a subdirectory of node_modules

1. Clone tree-sitter-cpp at the right revision
2. cd tree-sitter-cpp
3. npm install
4. Use the tree-sitter-c path from node_modules

Reply:
Tried your method, but 4. doesn't exist. I can't find any path in `node_modules`